test/object-storage.test.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var assert = require('assert');
+
+describe('object-storage connector', function() {
+  it('should be set up as loopback-component-storage', function(done) {
+    this.timeout(10000);
+    var app = require('./fixtures/object-storage-app');
+    var booted = false;
+    app.on('booted', function() {
+      if (booted) return;
```

Comment:
On the second thought, I think using `loopback-boot` is an overkill in this scenario. Also in general, the tests in LoopBack should not be relying on loopback-boot, but call LoopBack APIs directly.

Here is how you can simplify your test:

```js
it('should be set up as loopback-component-storage', () => {
  const app = loopback({localRegistry: true});
  app.dataSource('My-Object-Storage', {
    username: '0',
    // ...
    connector: 'object-storage',
  });

  // asserts
});
```